### PR TITLE
VACMS-5372 add favicon

### PR DESCRIPTION
**Pull request by tjheffner**
## Description

Relates to #5372 / ~#2746~. Also this related PR has been floating around since May: #5373 ... I created this new branch to avoid any potential merge conflicts with config / add the icon to the claro theme, but I could do this work in that PR too if that's more appropriate. I figured it could be safely closed instead :shrug:

- Adds favicon
- Removes environment_indicator addition to favicon
- Adds a screenshot for vagovclaro (seen on the appearance page)

## Testing done
Verified favicon showed as expected logged in and logged out with both vagovadmin and vagovclaro set as the default & admin themes.

## Screenshots
Screen Shot 2021-12-21 at 2 52 15 PM

Screen Shot 2021-12-21 at 11 35 19 AM

## QA steps
open PR environment, see that favicon is present
log in and change theme to vagovclaro, see that favicon is present for both themes

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [x] Code Quality Tests have passed.
- [x] Acceptance Criteria in related issue are met.
- [x] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Platform CMS team`
- [x] `Sitewide CMS Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [x] No announcement is needed for this code change.
  - [x] Merge & carry on unburdened by announcements

Personally I don't think an announcement is needed for a favicon update... just a nice quality of life update they may notice on their own?

### CODE REVIEW

**Comment by swirtSJW** [comment]
@tjheffner  this looks good but I want to check on one thing.
I think we want the cms brand to differentiate from va.go so tabs are identifiable.  This PR would make them the same
image
I know @kevwalsh  whipped up a quick one in this PR, but he never took  it off draft.
https://github.com/department-of-veterans-affairs/va.gov-cms/pull/5373

We should check with him first.

**Comment by kevwalsh** [comment]
This was [the reversed version of the VA logo](https://raw.githubusercontent.com/kevwalsh/va.gov-cms/VACMS-5373-favicon/docroot/themes/custom/vagovadmin/images/icons/favicon.png) that i [had prototyped](https://github.com/department-of-veterans-affairs/va.gov-cms/pull/5373/files#diff-d46b8ff683d80f4a82b0634e8a7365178e9dace9d8c39cca009dc1da845eca3c) to appear as the favicon. It looks like i hadn't taken the next step of creating an .ico from it, or downsizing it from 73kb. 

But yeah, the intent was to make the CMS look different than VA.gov. It seems OK per the [VA Tier 1 graphic standards](https://www.va.gov/opa/publications/graphicstandards/va_graphicstandardsguide_508_0113.pdf) 

It seems like there's an opportunity to replace the admin menu Druplicon with it too, in the short term. Not sure how many of our users know that the druplicon takes them back to the home page.

**Comment by rachel-kauff** [comment]
@kevwalsh @tjheffner What about something subtle that suggests "editing?" Like so:

Screen Shot 2021-12-28 at 9 17 52 AM

**Comment by rachel-kauff** [comment]
To scale:

Screen Shot 2021-12-28 at 9 41 35 AM

**Comment by tjheffner** [comment]
Updated favicon & druplicon in the toolbar to use reverse logo.

Used imagemagick to resize the 1874x1874px png logo to a 128x128png for the toolbar and a 32x32 ico for the favicon. 5.82kb and 4.29kb respectively vs 73kb.

Screen Shot 2021-12-28 at 11 51 06 AM
Screen Shot 2021-12-28 at 11 51 11 AM

**Comment by tjheffner** [comment]
@rachel-kauff I didn't use the dashed border favicon because i was worried it felt a little cluttered in both places but can definitely use that instead if it's a make or break thing

**Comment by rachel-kauff** [comment]
@tjheffner no worries, it was just an idea. And yeah, that wouldn't look good in the toolbar.

File: docroot/themes/custom/vagovclaro/assets/scss/components/_toolbar.scss [hunk 1]
```diff
@@ -14,6 +14,19 @@ body.toolbar-fixed {
...
+// replace druplicon with VA icon in the toolbar
+.toolbar-icon-9 .toolbar-icon-admin-toolbar-tools-help::before {
+  background-image: url('../../images/logo.png');
+}
+
+.toolbar-tray-horizontal .menu-item--expanded {
+  background-color: var(--color-white);
+}
+
+.toolbar-tray-horizontal .menu-item:hover {
```
Comment (kevwalsh) [inline docroot/themes/custom/vagovclaro/assets/scss/components/_toolbar.scss:26]: I love all this. 

Looks like you need an `:active` state too, there's a little druplicon flash when you click the VA logo.

**Review by kevwalsh — CHANGES_REQUESTED** [review]
I love all this.

File: docroot/themes/custom/vagovclaro/assets/scss/components/_toolbar.scss (hunk 1 above)
Comment (kevwalsh) [reply docroot/themes/custom/vagovclaro/assets/scss/components/_toolbar.scss:26]: Minor detail but i wonder if a transparent png would look better for the hover behavior. 

Cursor_and_kevin_walsh___VA_gov_CMS

Comment (tjheffner) [reply docroot/themes/custom/vagovclaro/assets/scss/components/_toolbar.scss]: Active state accounted for now, toolbar logo also made transparent 👍

**Review by kevwalsh — APPROVED** [review]
Looks great!

**Pull request merged**